x-pack/plugins/security_solution/public/resolver/view/panels/node_events_of_type.tsx

```diff
@@ -218,7 +215,7 @@ const NodeEventsOfTypeBreadcrumbs = memo(function ({
           text: (
             <FormattedMessage
               id="xpack.securitySolution.endpoint.resolver.panel.relatedEventList.countByCategory"
-              values={{ count: eventsInCategoryCount, category: eventType }}
+              values={{ count: eventsInCategoryCount, category: eventCategory }}
```

Comment:
ℹ️ Thanks for picking out all these places where it was still called `Type`. That was bound to get nasty and confusing.